slides/lesson1/solution/main.tf

```diff
@@ -40,29 +40,35 @@ resource "aws_security_group" "web" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
-  tags = "${var.tags}"
+  tags = var.tags
 }
 
 resource "aws_instance" "web" {
-  ami           = "${data.aws_ami.amazon_linux.id}"
+  ami           = data.aws_ami.amazon_linux.id
   instance_type = "t2.micro"
```

Comment:
Let's keep the instances consistent across our lessons. Which instance do you think we should use?
- t3.medium
- t3.large
- m5.large
cc @dancortes94 @chitopunk

Reply:
Vote on `t3.medium`.

Reply:
move this to m4/5.large and spot.

Reply:
Looks like we cannot use a spot with the `aws_instance` resource. We'd need to modify the solution using a [spot instance resource](https://www.terraform.io/docs/providers/aws/r/spot_instance_request.html)